Project.toml

```diff
@@ -7,3 +7,4 @@ version = "0.1.0"
 CxxWrap = "1f15a43c-97ca-5a2a-ae31-89f07a497df4"
 ROOT = "1706fdcc-8426-44f1-a283-5be479e9517c"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
This should only be in the extra section for tests

Reply:
I missed it, my bad. Resolved!